scripts/functions.sh

```diff
@@ -251,7 +251,7 @@ function cleanup {
       copyOutput
     elif [ "$copy" = "prompt" ]; then
       while true; do
-        read -rp "Copy the capture output locally ?" yn
+        read -rp "Copy the capture output locally? [yes/no] " yn
```

Comment:
Should we also manage the fully written word "yes" and "no" then ?

Reply:
If I'm correct, full yes/no already works (anything starting with a y or with a n works)
I wrote that based on the text below that already says:
"*) echo "Please answer yes or no." ;;"

Reply:
Indeed it already works :ok: